src/Host/Client/Impl/Definitions/IRCallbacks.cs

```diff
@@ -73,5 +73,8 @@ public interface IRCallbacks {
         /// </summary>
         /// <returns></returns>
         void ViewObject(string expression, string title);
+
+        void PackagesInstalled();
```

Comment:
Do we really need 2 callbacks? There are no parameters, so I'd rather use one `PackagesChanged`